src/CMakeLists.txt

```diff
@@ -15,20 +16,23 @@ ADD_LIBRARY(redisai_obj OBJECT
 IF(BUILD_TF)
     ADD_LIBRARY(redisai_tensorflow_obj OBJECT
             backends/tensorflow.c
+            backends.c
```

Comment:
I think we shouldn't add this file to the individual backends. Let's keep it only on the redisai.so side.
Let's just have `parseDeviceStr` in a `backends/util.h` file.